src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -85,6 +86,7 @@ Profile::Profile(const winrt::guid& guid) :
     _useAcrylic{ false },
     _scrollbarState{},
     _closeOnExit{ true },
+    _convertPasteLineEndings{},

```

Comment:
As a tangential change, do we want the WSL profiles that are auto-generated to default to having this setting set to true?

Reply:
I think that sounds like a good idea.